server/services/database/profiles.js

```diff
@@ -3,14 +3,39 @@ const config = require("../../config");
 const pool = new Pool(config);
 
 const getAllProfiles = () => {
-	return pool.query("SELECT id, first_name, last_name, phone_number, email FROM profiles")
-	.then((result) => result.rows)
-	.catch((error) => {
-		console.error(error);
-		throw error;
-	});
+	return pool
+		.query(
+			"SELECT id, first_name, last_name, phone_number, email FROM profiles",
+		)
+		.then((result) => result.rows)
+		.catch((error) => {
+			console.error(error);
+			throw error;
+		});
```

Comment:
```suggestion
```
We can remove this `catch` since we are `catch`ing higher up in `server/api/profiles.js`